torch/optim/lr_scheduler.py

```diff
@@ -95,12 +97,12 @@ def load_state_dict(self, state_dict):
         """
         self.__dict__.update(state_dict)
 
-    def get_last_lr(self):
+    def get_last_lr(self) -> List[float]:
         """ Return last computed learning rate by current scheduler.
         """
         return self._last_lr
 
-    def get_lr(self):
+    def get_lr(self) -> float:
```

Comment:
This should be `List[float]`, no?

Reply:
Yes, good catch. There may be more issues, the workflow here is to move all the existing annotations inline in one PR, add an `ignore_errors = True` in `mypy.ini`, and then in a second PR go through and fix all the errors. I'm only doing the former here, therefore I didn't review all annotations in detail.

Now that you've found this one I'll push a fix though.